types/setting.go

```diff
@@ -344,6 +348,28 @@ var (
 		ReadOnly:    false,
 		Default:     "10",
 	}
+	SettingDefinitionVolumeAttachmentDeletion = SettingDefinition{
+		DisplayName: "Volume Attachment Deletion Behavior",
+		Description: "Controls when Longhorn should delete a volume attachment on a failed node",
+		Category:    SettingCategoryDangerZone,
```

Comment:
Put it in `CategoryGeneral`. The options in the Danger Zone will crash all the volumes in the system.